src/message.rs

```diff
@@ -758,32 +918,32 @@ pub fn markseen_msgs(context: &Context, msg_ids: &[u32]) -> bool {
     if send_event {
         context.call_cb(Event::MsgsChanged {
             chat_id: 0,
-            msg_id: 0,
+            msg_id: MsgId::new(0),
```

Comment:
maybe generally use `MsgId::new_unset()` instead of the magic 0? see also my `is_unset` suggestion.

Reply:
From deltachat.h for DC_EVENT_MSGS_CHANGED: "data2 (int) msg_id for single added messages"

I think in many places 0 is used to signify multiple messages have changed, which -if you turn your head into UNIX/C style worthsmitthing- is also what the docs say I think.  That makes `MsgId::new_unset()` not really well suited for these.  I guess the problem is that the original API has, if I've been counting correctly, 3 different meanings of what msg_id=0 is.